src/main/scala/com/databricks/labs/overwatch/utils/Upgrade.scala

```diff
@@ -896,14 +897,29 @@ object Upgrade extends SparkSessionWrapper {
     println(upgradeFinalizedMsg)
   }
 
-  def upgradeTo0605(etlDatabaseName: String): Unit = {
-    updateJobSchema(etlDatabaseName)
+  private def upgradeDeltaTables(qualifiedName: String): Unit = {
+    try {
+      val tblPropertiesUpgradeStmt =
+        s"""ALTER TABLE $qualifiedName SET TBLPROPERTIES (
+      'delta.minReaderVersion' = '2',
+      'delta.minWriterVersion' = '5',
```

Comment:
The upgrade is trying to avoid completely rebuilding the spark_events_bronze table just to drop a bad column that was created due to new fields being published.

Rebuilding a massive table is quite compute heavy -- so instead -- I am trying to use a new functionality [column mapping](https://docs.databricks.com/delta/delta-column-mapping.html) but it REQUIRES 10.3+ and I'm telling customers they need to upgrade to 10.4LTS everywhere permanently. They won't be able to write to this again with a lower version of DBR since we'll have column mappings in the underlying delta_log. Since I'm doing an alter table out of the box I have to manually upgrade this metadata since there may not be any writes with 10.4LTS before this upgrade.